src/components/Tabs/Tabs.tsx

```diff
@@ -17,60 +17,69 @@ interface TabType {
 // prettier-ignore
 export interface TabsCustomProps {
     tabs : TabType[];
+    /** additional item to render inside the nav along with tab labels */
+    additionalNavItem?: React.ReactNode; 
 }
 
 export type TabsElementType = HTMLDivElement;
 export type TabsProps = Omit<CommonAndHTMLProps<TabsElementType>, keyof TabsCustomProps> & TabsCustomProps;
 
-export const Tabs = React.forwardRef(({ tabs, ...props }: TabsProps, ref: React.Ref<TabsElementType>) => {
-    const [activeTab, setActiveTab] = React.useState<TabType | undefined>(tabs.length > 0 ? tabs[0] : undefined);
-    const [isExiting, setIsExiting] = React.useState<boolean>(false);
+export const Tabs = React.forwardRef(
+    ({ tabs, additionalNavItem, ...props }: TabsProps, ref: React.Ref<TabsElementType>) => {
+        const [activeTab, setActiveTab] = React.useState<TabType | undefined>(tabs.length > 0 ? tabs[0] : undefined);
+        const [isExiting, setIsExiting] = React.useState<boolean>(false);
 
-    const handleTabChange = (tab: TabType) => {
-        setIsExiting(true);
-        setTimeout(() => {
-            setIsExiting(false);
-            setActiveTab(tab);
-        }, 120);
-    };
+        const handleTabChange = (tab: TabType) => {
```

Comment:
Install and use `prettier`, fren!

Reply:
prettier only suggested these formatting changes 😄